### PR TITLE
Fix typo and group by sample

### PR DESCRIPTION
**Pull request by ihostage**

### CODE REVIEW

**Review by ignasi35 — DISMISSED** [review]
Nice fix: keep things grouped and tidy! :-)

File: .travis.yml [hunk 1]
```diff
@@ -25,16 +25,16 @@ jobs:
...
-    - "bin/runSbtJob shopping-cart/shopping-cart-java test"
-    name: "Run tests gRPC example Java"
+    - "bin/runSbtJob shopping-cart/shopping-cart-scala test"
+    name: "Run tests Shopping Cart example Scala"
 
   - script:
-    - "bin/runSbtJob grpc-example/grpc-example-scala test"
-    name: "Run tests gRPC example Scala"    
+    - "bin/runSbtJob shopping-cart/shopping-cart-java test"
+    name: "Run tests Shopping Cart example Java"
```
Comment (ignasi35) [inline .travis.yml:33]: Shopping cart seems to break the convention to put Java before Scala.

**Review by ignasi35 — DISMISSED** [review]
just noticed a small detail

File: .travis.yml (hunk 1 above)
Comment (ihostage) [reply .travis.yml:33]: My mistake 😞 Fixed 😄

**Comment by octonato** [comment]
Oh, how could I be so distracted? Thanks @ihostage. Good to have you around always paying attention on what we do. ;-)

**Review by octonato — APPROVED** [review]

**Pull request merged**